body-pix/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tensorflow-models/body-pix",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
Update all these models to 1.0.0 since they now expect tfjs 1.0 as peer and most users that depend on 0.x of these models, have tfjs 0.x as their dep